examples/README.md

```diff
@@ -0,0 +1,97 @@
+# wolfSSL examples directory
+#### client and server
+
+ These directories contain a client (client.c) and server (server.c) that utilize a variety of the wolfSSL library's capabilities. The manner in which both programs operate can depend on the configure or can be specified at run-time depending on the end goal. Both applications contain testing as well as benchmarking code.
+
+ ```
+ Compile
```

Comment:
Please move the "Compile" out of the triple tick. You can add sh to the triple tick to inform mark-down of format. Same for the other triple tick sections... the title should not be in the shell block.